test/entanglement_forging/test_entanglement_forging_knitter.py

```diff
@@ -70,6 +72,7 @@ def create_mock_ansatz_circuit(self, num_qubits: int) -> QuantumCircuit:
 
         return ansatz
 
+    @pytest.mark.slow
```

Comment:
I think this one should run in a second or two. I'd be in favor of leaving this test available in every run, just as a sanity check on 80+% of the EF source code.

Reply:
Yeah, I should have actually timed these.  I added a timing report in ed173849a3f1e9dbbe19da1ada4d13d508ffaaa2.

Here's what it shows on my current machine:

```
============================= slowest 10 durations =============================
12.03s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_asymmetric_bitstrings_CN
1.67s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_asymmetric_bitstrings_O2
1.13s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_asymmetric_bitstrings_CH3
0.88s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_entanglement_forging_H2O
0.69s call     test/circuit_cutting/test_circuit_cutting.py::TestCircuitCutting::test_circuit_cutting_manual
0.62s call     test/entanglement_forging/test_entanglement_forging_ground_state_solver.py::TestEntanglementForgingGroundStateSolver::test_entanglement_forging_vqe_hydrogen
0.13s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_entanglement_forging_H2
0.02s call     test/entanglement_forging/test_entanglement_forging_knitter.py::TestEntanglementForgingKnitter::test_entanglement_forging_driver_H2
```

I enabled the three fastest of these in 27bc0f01bb3e74da6ce53cfa2140f3e8dcba9095.

Reply:
Could we mark O2 and CN as slow and let everything else run? The rest each test slightly different things, so it would be great to have them run in the default as well. CN and O2 are actually redundant and truly don't *need* to be run. If those fail, the CH3 test should fail

Reply:
I made these changes, and actually, now coverage reports the same lines covered regardless of whether the slow tests are run.  With this in mind, I decided to flip it such that the slow tests are skipped by default.  All tests are run on CI, or if someone runs `--run-slow` explicitly.  I've also modified it so the report at the end of a test run says _why_ certain tests were skipped:

```
SKIPPED [1] test/circuit_cutting/test_circuit_cutting.py:52: cplex is not installed
SKIPPED [1] test/entanglement_forging/test_entanglement_forging_knitter.py:306: skipping slow test, as --run-slow was not provided
SKIPPED [1] test/entanglement_forging/test_entanglement_forging_knitter.py:231: skipping slow test, as --run-slow was not provided
================== 6 passed, 3 skipped, 453 warnings in 4.48s ==================
```

Reply:
> I made these changes, and actually, now coverage reports the same lines covered regardless of whether the slow tests are run. With this in mind, I decided to flip it such that the slow tests are skipped by default. All tests are run on CI, or if someone runs `--run-slow` explicitly. I've also modified it so the report at the end of a test run says _why_ certain tests were skipped:
> 
> ```
> SKIPPED [1] test/circuit_cutting/test_circuit_cutting.py:52: cplex is not installed
> SKIPPED [1] test/entanglement_forging/test_entanglement_forging_knitter.py:306: skipping slow test, as --run-slow was not provided
> SKIPPED [1] test/entanglement_forging/test_entanglement_forging_knitter.py:231: skipping slow test, as --run-slow was not provided
> ================== 6 passed, 3 skipped, 453 warnings in 4.48s ==================
> ```

Awesome, I realized I was saying "default" before, when I really meant "fast tests only". Glad we can make "fast tests only" the default now :). I suspected CN, CH3, and O2 were testing the exact same pipeline -- glad the coverage backs that up.